### PR TITLE
FAQ Page + folder restructure

### PR DESCRIPTION
**Pull request by daniaalnahdi**
Note: I included the CSS within the components since they're so minimal, but can refactor if need be

ezgif-2-405bb6c0c5b3

ezgif-2-be8fa09a47f7

### CODE REVIEW

File: src/components/FAQSection.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { Accordion, Card } from 'react-bootstrap';
+
+const styles = {
```
Comment (feliciazhang) [inline src/components/FAQSection.js]: I think this is still better off in a css file, you don't need to make a new one though feel free to pop it in like index.css or something

File: src/components/MainNavbar.js [hunk 2]
```diff
@@ -21,6 +21,10 @@ class MainNavbar extends React.Component {
             <NavLink className="header-text">Open Projects</NavLink>
           </LinkContainer>
 
+          <LinkContainer to="/faq">
+            <NavLink className="header-text">FAQ</NavLink>
+          </LinkContainer>
+
```
Comment (nupur-n) [inline src/components/MainNavbar.js]: small thing - thoughts on having the FAQ as the last item in the nav?

**Review by nupur-n — DISMISSED** [review]

File: src/pages/FAQPage.js [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+        selectedFAQ = null;
+    }
+
+    this.setState({ faq: selectedFAQ, title: innerHTML });
+  };
+
+  render() {
+    return (
+      <div className="container mt-5">
+        <h1>Frequently Asked Questions</h1>
```
Comment (feliciazhang) [inline src/pages/FAQPage.js]: thoughts on changing this to just say FAQs? feels weird to see it written out for some reason lol

File: src/pages/FAQPage.js [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+          <Col md={4} style={styles.ButtonCol}>
+            <Button
+              value="client"
+              className="button"
+              variant="warning"
+              size="md"
+              style={styles.Button}
+              onClick={this.handleClick}
+            >
+              Requesting a Project
```
Comment (feliciazhang) [inline src/pages/FAQPage.js:59]: I _think_ the rest of the site uses sentence casing (only first word is capitalized) so we should keep consistent- ignore this if that isn't true

File: src/components/MainNavbar.js [hunk 5]
```diff
@@ -21,6 +21,10 @@ class MainNavbar extends React.Component {
             <NavLink className="header-text">Open Projects</NavLink>
           </LinkContainer>
 
+          <LinkContainer to="/faq">
```
Comment (nupur-n) [inline src/components/MainNavbar.js]: thoughts on making this the last one in the nav bar?

File: src/containers/SiteContainer.js [hunk 6]
```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import "../styling/site-container-styling.css";
```
Comment (feliciazhang) [inline src/containers/SiteContainer.js]: we can move this import to the top of the index.css file

**Review by feliciazhang — COMMENTED** [review]
looks really good!!

File: src/components/FAQSection.js [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+  },
+  Accordion: {
+    margin: '1em 0'
+  },
+  Question: {
+    color: 'black',
+    cursor: 'pointer'
+  },
+  Answer: {
+    color: 'grey',
```
Comment (feliciazhang) [inline src/components/FAQSection.js]: there's a specific css var for a grey `var(--main-grey)` not sure if it's the same as what you're looking for

File: src/components/MainNavbar.js (hunk 2 above)
Comment (liu-caro) [reply src/components/MainNavbar.js]: I'm a fan of this idea!

**Comment by liu-caro** [comment]
general comment: could we have the initial state be "clicked" into the first section? I'm not sure if i'm a fan of an empty screen/could help people understand to click into the sections better. looks great though!

File: src/containers/SiteContainer.js [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+              </div>
+            )}
+          />
+        </Router>
+      </div>
+    );
+  }
+}
+
+export default SiteContainer;
```
Comment (liu-caro) [inline src/containers/SiteContainer.js]: What is the purpose of this SiteContainer and do we really need it? I'm pretty sure I removed this page and refactored all of the routing to go into App.js so this page isn't necessary anymore - maybe you were just refactoring from an outdated dev branch? If so, could you delete this file please?

Comment (nupur-n) [reply src/containers/SiteContainer.js]: Yes looks like the outdated container, @daniaalnahdi if you delete the file it should not cause any issues on the site!

**Review by nupur-n — APPROVED** [review]
Looks good to me, just the comment about the outdated Site Container. Love the overall look and feel of this!

File: src/index.css [hunk 9]
```diff
@@ -21,3 +21,19 @@ code {
...
+}
+
+.faq-button-selected {
+  background-color: '#ffc107',
+}
+
+.faq-accordion {
+  cursor: pointer;
+  color: black;
+}
```
Comment (liu-caro) [inline src/index.css]: I'm going to move this to a separate file, just so it's cleaner!

**Review by liu-caro — APPROVED** [review]

**Pull request merged**